src/main/resources/assets/additionalguns/models/item/m4a1s.json

```
@@ -1,5 +1,5 @@
 {
-	"credit": "Made with Blockbench. Made by VIKTORWOOD",
+	"credit": "Made with Blockbench",
```

Same goes for this one :)
```suggestion
	"credit": "Made with Blockbench. Made by VIKTORWOOD",
```